pkg/cluster/certs.go

```diff
@@ -66,12 +66,21 @@ func (c *Controller) RotateAllCerts(ctx context.Context) error {
 		c.Log.Warnf("Failed to delete rotate pods: %v", err)
 	}
 	opts := metav1.ListOptions{
-		LabelSelector: PrimaryRoleLabel,
+		LabelSelector: "node-role.kubernetes.io/master=",
 	}
 	node, err := c.Config.Client.CoreV1().Nodes().List(context.TODO(), opts)
 	if err != nil {
 		return errors.Wrap(err, "list primary nodes")
 	}
+	if len(node.Items) == 0 {
```

Comment:
interestingly if wrong label is looked up, `err is still nil` and the node list is empty. so checking for len of node.Items